lib/msal-browser/README.md

```diff
@@ -38,26 +38,25 @@ See [here](./FAQ.md).
+| Date | Release | Announcement | Main features |
+| ------| ------- | ---------| --------- |
+|  | @azure/msal-browser v2.0.0-beta | No release notes yet | Beta version of the `@azure/msal-browser` package; relies on `@azure/msal-common` v1.0.0-beta |
+| January 17, 2020 | @azure/msal-browser v2.0.0-alpha | No release notes yet | Alpha version of the `@azure/msal-browser` package with authorization code flow for SPAs working in dev; relies on msal-common v1.0.0-alpha |
 
 ## Prerequisites
 
-- `msal-browser` is meant to be used in [Single-Page Application scenarios](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-overview).
+- `@azure/msal-browser` is meant to be used in [Single-Page Application scenarios](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-overview).
 
-- Before using `msal-browser` you will need to [register an application in Azure AD](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to get a valid `clientId` for configuration, and to register the routes that your app will accept redirect traffic on.
+- Before using `@azure/msal-browser` you will need to [register an application in Azure AD](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to get a valid `clientId` for configuration, and to register the routes that your app will accept redirect traffic on.
 
-- You will also need a CORS-disabled browser until the AAD service has rolled out changes to introduce CORS requests on the `/token` endpoint. This is not a recommended production setting, but for the purposes of this alpha you should follow these instructions:
-    - We recommend using Chrome for this. 
-        - For Windows machines, you can do the following:
-            1. Right click on your desktop -> New -> Shortcut
-            2. Paste the following: 
-            ```javascript
-            "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --disable-web-security --disable-gpu --user-data-dir=~/chromeTemp
-            ```
-            3. Click next and create a name for the shortcut (i.e. Chrome-no-cors)
-        - For other machines, you can follow the steps [here](https://alfilatov.com/posts/run-chrome-without-cors/) to figure out how to run Chrome withput CORS enabled for your OS.
+- Once you have registered your application, you will need to do two things in order to ensure `@azure/msal-browser` will successfully retrieve tokens:
+    1. Change your redirect URI type to enable CORS. You can do this by going to the manifest editor for your app registration in the portal, finding the `replyUrlsWithType` section and changing the type of your redirect URI to `SPA`. This may remove the affected redirect URIs from the Web platform Authentication tab - that's OK! We are working on getting UI set up for this.
```

Comment:
I think `spa` may be lowercase? I'm not sure if it matters but can you double check on this?

Reply:
I have changed it to uppercase and the manifest changes it to "Spa" so I think it is case insensitive